smarts/core/smarts.py

```diff
@@ -88,6 +88,7 @@ def __init__(
         timestep_sec=0.1,
         reset_agents_only=False,
         zoo_workers=None,
+        authkey=b"secret",
```

Comment:
```suggestion
        auth_key=None,
```

Reply:
Changed default value of auth_key of type byte string to be b"None".